src/main/java/gregtech/integration/exnihilo/recipes/ExNihiloRecipes.java

```diff
@@ -0,0 +1,234 @@
+                .outputs(new ItemStack(Blocks.STONE, 1, 5))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 0))
+                .outputs(OreDictUnifier.get(stone, Basalt, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 1))
+                .outputs(OreDictUnifier.get(stone, GraniteBlack, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 2))
+                .outputs(OreDictUnifier.get(stone, Marble, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 3))
+                .outputs(OreDictUnifier.get(stone, GraniteRed, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+
+        // Crushed stones
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .input(Blocks.SAND)
+                .output(ModBlocks.dust)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(Blocks.STONE, 1, 1))
+                .output(ModBlocks.crushedGranite)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(Blocks.STONE, 1, 3))
+                .output(ModBlocks.crushedDiorite)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(Blocks.STONE, 1, 5))
+                .output(ModBlocks.crushedAndesite)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .input(Blocks.NETHERRACK)
+                .output(ModBlocks.netherrackCrushed)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .input(Blocks.END_STONE)
+                .output(ModBlocks.endstoneCrushed)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+
+        // Meshes
+        if (ExNihiloConfig.harderMeshes) {
+            ModHandler.removeRecipeByName("exnihilocreatio:item_mesh_2");
+            ModHandler.addShapedRecipe("bronze_mesh", new ItemStack(ModItems.mesh, 1, 2), "TST", "STS", "TST",
```

Comment:
Since this is a flint stiffened mesh, I feel like there should at least be a piece of flint in the recipe, maybe in the top middle?